api/src/main/java/org/datacleaner/api/ExecutionMeteringMessage.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.api;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Added by jakub on 22.10.15
+ */
+public class ExecutionMeteringMessage implements ComponentMessage {
```

Comment:
"Execution" seems a bit irrelevant in this regard because everything is execution. I would rather use words that really pertain to this, or none at all. For instance: UsageMeteringMessage, ServiceMeteringMessage, ComponentUsageMessage?